tedge/src/cli/disconnect/disconnect_bridge.rs

```diff
@@ -70,6 +70,23 @@ impl DisconnectBridgeCommand {
             self.service_manager()
                 .stop_and_disable_service(self.cloud.dependent_mapper_service(), std::io::stdout());
         }
+        match self.cloud {
+            Cloud::C8y => {
+                if which("tedge_agent").is_err() && which("tedge_mapper").is_err() {
+                    println!(
+                        "Warning: tedge_mapper is not installed. We recommend to install it.\n"
```

Comment:
Same as above here.
New message:
"Software management is not installed. Skipping disabling related components."

Reply:
I have updated these things in the recent commits, please have a look.

Reply:
I feel we do not need to mention about disabling software management service, we can just hide this.